website/blog/2020-09-01-63updates.md

```diff
@@ -0,0 +1,118 @@
+
+Sometimes it really is the little things that make a huge difference and in this case Autolinking is no exception!
+
+Now when you install new modules and build your app we will automatically hook-up all those necessary native platform-specific code pieces for you in Windows. This will save you time and (possibly) a headache, if C++ isn't your strong suit.
+
+We've also provided some new documentation around how to use [Autolinking with Native Modules](https://microsoft.github.io/react-native-windows/docs/native-modules-autolinking) in Windows.
+
+# PlatformColor
+
+![platformcolor-windows](assets/platformcolor_demo.gif)
+
+`PlatformColor` is a new API React Native core introduced that gives React Native app components the ability to leverage native system colors - meaning you can now subscribe to certain colors or brushes in your `StyleSheet` that respond to native platform actions like theme switching, user accent colors, or high contrast changes.
+
+Head over to the [official blog post](https://reactnative.dev/blog/2020/07/06/version-0.63#native-colors-platformcolor-dynamiccolorios) about this API to learn more about accessing native colors across all platforms, or check out our [new guidance documentation on PlatformColor](https://microsoft.github.io/react-native-windows/docs/windowsbrush-and-theme) to learn how to take advantage of Windows-specific colors and materials in your app.
+
+> Since the release of this API, the older Windows `AppTheme` API will be deprecated in favor of `PlatformColor`.
+
+# Pressable
+
+[Pressable](https://reactnative.dev/docs/pressable) is now available on React Native for Windows apps as well. This new core API helps make customizing your interaction experiences more robust and easier to fit around your custom components and scenario needs.
+
+# Community Modules
+
+As with every big release, we're still working hard at adding Windows implementations to important community modules.
+
+
+Come check out what we've enabled for you this release!
+
+## react-navigation
+
+
+![reactnavigation-rnwindows](assets/react-nav.gif)
+
+One of the more popular and much needed [react-navigation](https://github.com/react-navigation/react-navigation/tree/main/packages/native) module is now supported on Windows. Navigate between screens with native ease and preserve your stack history in this amazing community module.
+
+
+## react-native-checkbox
+
+
+![checkbox-rnwindows](https://raw.githubusercontent.com/react-native-community/react-native-checkbox/master/screenShots/demo-windows.png)
+
+The community module [react-native-checkbox](https://github.com/react-native-community/react-native-checkbox) has also received Windows support this release. CheckBox is a rather unique native control for us and we're happy to have it in.
+
+Special shoutout and thank you to the [community members who helped make this possible!](https://github.com/react-native-community/react-native-checkbox/pull/68)
+
+# The Hermes Engine

```

Comment:
Just realized, there is also the new experimental feature to consume RNW as a nuget package. Docs are here - https://microsoft.github.io/react-native-windows/docs/next/nuget 

It might be useful to make this section about "Experimental features" with some words on why they are experimental and include both Hermes and RNW as nuget package in here with pointers to the docs.

Reply:
actually it looks like the experimental sidebar section was merged in a separate PR - so I don't think I'll edit it here, but we can add it after this one is merged.